src/pages/loading-waiting-states/progress-bar/index.tsx

```diff
@@ -156,14 +149,18 @@ export const ProgressBar = (): JSX.Element => {
         if (status === 'Complete') {
             return;
         }
+        setFileUploadList((oldList) => oldList.map((item) => (item.id === id ? { ...item, open: false } : item)));
+    }, []);
+
+    const handleExited = useCallback((id: number) => {
         setFileUploadList((oldList) => oldList.filter((item) => item.id !== id));
     }, []);
 
     const handleRequestClose = useCallback((event: any, reason: string, id: number) => {
```

Comment:
some of these callback names are a little confusing. What does handleRequestClose versus handleExited mean? removeListItem doesn't actually remove items from the list.

I would like to see these named something perhaps a bit more descriptive like:
`markUploadComplete` => sets the open property to false (currently removeListItem)
`removeFileFromList` => deletes the file from the list (currently handleExited)

What is the difference between removeListItem and handleRequestClose? They seem to be doing the same thing...could they be combined?